### DISCUSSION TITLE
Play attach mp3/audio files

### DISCUSSION THREAD

**Discussion by vinbrule** [ViennaRSS/vienna-rss#1319] [Ideas]
Hi,

Many language learning sites provide RSS with link to mp3/audio. It would be wonderful that Vienna support this. 

PS: Also, I am new to Xcode but willing to help. Please tell me some starting points to do it
edit: play mp3 inside vienna

**Comment by barijaona** [comment]
Vienna is able to read [enclosures in RSS 2](https://en.wikipedia.org/wiki/RSS_enclosure) and in Atom feeds.
I'll have to check if mod_enclosure for RSS 1.0 is also supported.

From my point of view, the real problem is that too many sites tend to hide their RSS feeds. But they do provide them to Apple Podcasts, Spotify and so on !

**Comment by vinbrule** [comment]
@barijaona thanks, i just update my description to make clear. I mean that vienna is able to play mp3 inside itself. No need to download and play later.
Bests
